sdk/monitor/Azure.Monitor.Query/api/Azure.Monitor.Query.netstandard2.0.cs

```diff
@@ -306,18 +317,7 @@ public partial class MetricResult
     public partial class MetricResultsResponse
     {
         internal MetricResultsResponse() { }
```

Comment:
Could we shorten the name? ResultsResponse seems redundant.

Reply:
Updated to MetricsBatchResult :) also matches Java naming